src/com/nicoles_change_maker/ChangeMaker.java

```diff
@@ -0,0 +1,61 @@
+package com.nicoles_change_maker;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Iterator;
+/**
+ * Created by bears8yourface on 1/5/16.
+ */
+public class ChangeMaker {
+    double remainingDollars;
+
+    public HashMap makeChange(double dollarAmount) {
```

Comment:
This method signature is also the source of a compiler warning, because the return type doesn't have type parameters. The Java compiler doesn't just want to know that this returns a `HashMap`, but what types of keys and values are inside. Is this a `HashMap<String, Integer>`, a `HashMap<Coin, Amount>`, a `HashMap<HashMap<String, String>,List<Integer>>`, or something even crazier? Your code will still compile without these annotations, but it's generally a good idea to include them.